docs/user/gcp/install_upi.md

```diff
@@ -98,6 +123,38 @@ del data["spec"]["privateZone"];
 open(path, "w").write(yaml.dump(data, default_flow_style=False))'
 ```
 
+```console
+spec:
+  baseDomain: example.com
+```
+
+### Update the cloud-provider manifest (optional)
+If you are installing into a [Shared VPC][sharedvpc],
+update the cloud provider configuration so it understands the network and subnetworks are in a different project (host project).
```

Comment:
The title/first section must mention this section is only for shared vpc xpn and other can just skip this.